index.js

```diff
@@ -62,17 +74,68 @@ module.exports = {
         */
         'no-loop-func': 'error',
 
+        /*
+            Enforce parenthesis in cases of mixed precedence.
+        */
+        'no-mixed-operators': ['error', {
+            groups: [
+                ['%', '**'],
+                ['%', '+'],
+                ['%', '-'],
+                ['%', '*'],
+                ['%', '/'],
+                ['/', '*'],
+                ['&', '|', '<<', '>>', '>>>'],
+                ['==', '!=', '===', '!=='],
+                ['&&', '||'],
+            ],
+            allowSamePrecedence: true
+        }],
+
         /*
             This rule defends against prototype pollution vulnerabilities that exploitable with
             XSS.
         */
         'no-prototype-builtins': 'error',
 
+        /*
+            Higher-order function composition is preferred for iterative loops. For...in statements
```

Comment:
The phrase "higher-order function composition" might not be immediately understood for some readers (cough...me...cough) so I think adding an example would help. Something like
"Higher-order function composition (array.forEach(...)) is preferred for iterative loops".

Or we could even provide explicit guidance by stealing the guidance from airbnb's readme.
    > Use `map()` / `every()` / `filter()` / `find()` / `findIndex()` / `reduce()` / `some()` / ... to iterate over arrays, and `Object.keys()` / `Object.values()` / `Object.entries()` to produce arrays so you can iterate over objects.